src/Diagnostics.ModelsAndUtils/Attributes/ResourceFilter/ContainerAppFilter.cs

```diff
@@ -6,4 +6,11 @@ public ContainerAppFilter(bool internalOnly = true) : base(ResourceType.Containe
         {
         }
     }
+
+    public class MicrosoftAppContainerAppFilter : ResourceFilterBase
+    {
+        public MicrosoftAppContainerAppFilter(bool internalOnly = true) : base(ResourceType.ContainerApp, internalOnly)
```

Comment:
Will this still work if its `ResourceType.ContainerApp`?

Reply:
I searched all the places that the ResourceType property is referenced. It seems to work.